apps/backoffice-v2/src/lib/blocks/hooks/useProcessTrackerBlock/useProcessTrackerBlock.ts

```diff
@@ -0,0 +1,23 @@
+import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { useEntityLogic } from '@/pages/Entity/hooks/useEntityLogic/useEntityLogic';
+import { useMemo } from 'react';
+
+export const useProcessTrackerBlock = () => {
+  const { workflow, plugins } = useEntityLogic();
+
+  const block = useMemo(
```

Comment:
Is there room for the process tracker to use the node cell instead? It seems like a new cell was created just so it could be rendered in the tabs. Let me know what you think.